config/config.json

```diff
@@ -21,7 +21,7 @@
         "EnablePostUsernameOverride": false,
         "EnablePostIconOverride": false,
         "EnableTesting": false,
-        "EnableDeveloper": false,
+        "EnableDeveloper": true,
```

Comment:
Can you revert all the changes to config.json except your addition of `MaxChannelsPerTeam`?